### PR TITLE
Added projects list file

### LINKED ISSUE #4: Create a list of costs associated with any project
As much as we would like to self-fund these efforts, I know some of us would also like to help out our families.  Here we can put in actual or estimated costs for the projects. We can decide later if we need to reach out to our networks to request donations to set any of the projects up. 

 We could also use this space to list what we already have (eg: Twilio credits).

### PR DESCRIPTION
**Pull request by froi**
Closes #4 

File to list projects and funding needs.

### CODE REVIEW

**Comment by chamakits** [comment]
Thought about maybe adding a note column, but that would probably get out of control large quickly. 

Lgtm

File: PROJECTS.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+# Where does the money go?
+
+The money goes directly to the owners of the project or the person that is paying for the infrastructure fees.
+
+# How can I add my project?
+
+* Clone this repo
+* Add your project following the format of the ones before.
+   * Add your current cost
```
Comment (rlconley) [inline PROJECTS.md:19]: If we are asking for a number, do we want to also ask for what the costs are specifically for, like hosting, diesel for generator, etc?

Comment (froi) [reply PROJECTS.md:19]: So like a detail of cost then?

**Review by miguelrios — APPROVED** [review]

File: PROJECTS.md (hunk 1 above)
Comment (rlconley) [reply PROJECTS.md:19]: That's what I was thinking. Doesn't have to be exact but can answer the question "What is my money paying for?"

Comment (AnaSylvia) [reply PROJECTS.md:19]: Yes. It should have details. If nothing else, for transparency when asking for funding.

Comment (froi) [reply PROJECTS.md:19]: Excellent.

**Pull request merged**